tests/unit/lms/services/canvas_api/conftest.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from lms.services.canvas_api import TokenStore
+from tests import factories
+
```

Comment:
Putting all of this out side of the test looks very strange, but they are re-used in later tests for the public client.